app/eventteam.html

```diff
@@ -94,7 +94,7 @@
                                         <md-switch ng-model="$parent.filterSkillsMatchSwitch" ng-change="filterTeams(filterPlacesSwitch, filterSkillsMatchSwitch)" ng-if="user">Match my skills</md-switch>
                                     </div>
                                     <div class="col-sm-3">
-
+                                        <md-switch ng-model="$parent.filterMissingSkillsMatchSwitch" ng-change="filterTeams(filterPlacesSwitch, filterMissingSkillsMatchSwitch)" ng-if="user">Match missing skills</md-switch>
```

Comment:
also change the other 2 filter switches `filterTeams() call` to `filterTeams(filterPlacesSwitch, filterSkillsMatchSwitch, filterMissingSkillsMatchSwitch)`